### ISSUE TITLE
Disabling log doesn't work

### ISSUE THREAD

**Issue by Sarke** [wagoodman/dive#168]
I am on the latest release `0.6.0` and the following still creates a `dive.log` file:

```yaml
log:
  enabled: false
```

However, this works:

```yaml
log:
  path: /dev/null
```

Great tool btw, love it.

**Comment by akshaychhajed** [comment]
@Sarke dive prints name of config used if it found one. 
```
dive image:latest
Using config file: /somelocation/.dive.yaml
Fetching image... (this can take a while with large images)
Parsing image...
Analyzing image...
Building cache...
```
Also if you are using `~/.config/dive.yaml` look at #170

**Comment by akshaychhajed** [comment]
Looks like this has been fixed in master (after 0.6.0 release)
https://github.com/wagoodman/dive/pull/148

**Comment by Sarke** [comment]
The config file is being read, that is not the issue.

**Comment by wagoodman** [comment]
@Sarke thanks! Glad you like it! You should see this fix in the 0.7.0 release (within the hour).
